### PR TITLE
feat: extend settings via configuration

### PR DESCRIPTION
**Pull request by amanji**
Signed-off-by: Akiff Manji <akiff.manji@gmail.com>

### CODE REVIEW

File: app/App.tsx [hunk 1]
```diff
@@ -24,9 +27,66 @@ import bcwallet from './src'
...
+    initStoredLanguage().then()
+  }, [])
+
   const [agent, setAgent] = useState<Agent | undefined>(undefined)
-  initStoredLanguage()
+  const { t } = useTranslation()
+  const { navigate } = useNavigation()
+
+  const settings = [
+    // TODO: Resolve pages for these settings
```
Comment (jleach) [inline app/App.tsx]: Able to to `//TODO:(AM)` so we know who's todo it is?

File: app/src/index.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { pages } from './screens/OnboardingPages'
 import Splash from './screens/Splash'
 import Terms from './screens/Terms'
 import { defaultTheme as theme } from './theme'
-const localization = _merge({}, translationResources, {
+const localization = merge({}, translationResources, {
```
Comment (jleach) [inline app/src/index.ts:15]: Not overly important but a space above here may help readability.

File: app/src/localization/en/index.ts [hunk 3]
```diff
@@ -33,6 +33,17 @@ const translation = {
   "NetInfo": {
     "NoInternetConnectionTitle": "No internet connection",
     "NoInternetConnectionMessage": "You're unable to access services using BC Wallet or receive credentials until you're back online.\n\nPlease check your internet connection."
+  },
```
Comment (jleach) [inline app/src/localization/en/index.ts:36]: Able to add these place holders in EN to Fr and Br?

File: app/src/screens/Splash.tsx [hunk 4]
```diff
@@ -34,9 +34,9 @@ import { Config } from 'react-native-config'
 import { SafeAreaView } from 'react-native-safe-area-context'
 import Toast from 'react-native-toast-message'
 
-interface SplashProps {
-  setAgent: React.Dispatch<React.SetStateAction<Agent | undefined>>
-}
+// interface SplashProps {
```
Comment (jleach) [inline app/src/screens/Splash.tsx]: Axe dead code?

**Review by jleach — CHANGES_REQUESTED** [review]

File: app/src/index.ts (hunk 2 above)
Comment (amanji) [reply app/src/index.ts:15]: The linter should automatically take care of formatting here. Not sure if it's not working?

File: app/src/localization/en/index.ts (hunk 3 above)
Comment (amanji) [reply app/src/localization/en/index.ts:36]: Already in Bifold. These override the en wording for BC Wallet. Which brings me to the question, does BC Wallet specifically have multi-language support? If so then we need to outright create translation files for fr and pt-br. If not, then we need to override the language screens.

**Review by jleach — APPROVED** [review]

**Pull request merged**